interfaces/cython/cantera/examples/reactors/NonIdealShockTube.py

```diff
@@ -90,33 +94,39 @@ def ignitionDelay(states, species):
 estimatedIgnitionDelayTime = 0.005
 t = 0
 
-counter = 1;
+counter = 1
 while(t < estimatedIgnitionDelayTime):
     t = reactorNetwork.step()
-    if (counter%20 == 0):
+    if (counter % 20 == 0):
         # We will save only every 20th value. Otherwise, this takes too long
         # Note that the species concentrations are mass fractions
         timeHistory_RG.append(r.thermo.state, t=t)
-    counter+=1
+    counter += 1
 
 # We will use the 'oh' species to compute the ignition delay
 tau_RG = ignitionDelay(timeHistory_RG, 'oh')
 
 # Toc
 t1 = time.time()
-print('Computed Real Gas Ignition Delay: {:.3e} seconds. Took {:3.2f}s to compute'.format(tau_RG, t1-t0))
+print(
+    "Computed Real Gas Ignition Delay: {:.3e} seconds. "
+    "Took {:3.2f}s to compute".format(tau_RG, t1-t0)
+)
 
 
 # Ideal gas IDT calculation
 # Create the ideal gas object:
-ideal_gas = ct.Solution('nDodecane_Reitz.cti','nDodecane_IG')
+ideal_gas = ct.Solution(
+    "nDodecane_Reitz.yaml", "nDodecane_IG", transport_model=None
+)
 
 # Set the state of the gas object:
 ideal_gas.TP = reactorTemperature, reactorPressure
 
 # Define the fuel, oxidizer and set the stoichiometry:
-ideal_gas.set_equivalence_ratio(phi=1.0, fuel='c12h26',
-                                oxidizer={'o2':1.0, 'n2':3.76})
+ideal_gas.set_equivalence_ratio(
+    phi=1.0, fuel='c12h26', oxidizer={'o2': 1.0, 'n2': 3.76}
+)
```

Comment:
In my opinion, this is much easier to read than the previous formatting (for reference, the previous formatting was:
```python
ideal_gas.set_equivalence_ratio(phi=1.0, fuel='c12h26',
                                oxidizer={'o2':1.0, 'n2':3.76})
```
)
The main reason I find it easier to read is that all of the arguments end up on the same line, so you can just read them across. This does separate things from the function name being called, so I can see the other side. @ischoegl Do you have an opinion here?

Reply:
@bryanwweber ... since you’re asking: I tend to write my own code in the style of the pre-existing format, as I personally don’t like single parentheses on a line (it simply doesn’t look like python code to me). At the same time, this is form and not function so it ultimately makes little difference: both are perfectly readable.

Reply:
I reverted the change that resulted in the closing paren on its own line, but retained adding the spaces after the colons in the dictionary.